k-distribution/include/kframework/builtin/README.md

```diff
@@ -0,0 +1,22 @@
+---
+copyright: Copyright (c) 2020 K Team. All Rights Reserved.
+---
+
+K Builtins
+==========
+
+The K Builtins (also referred to as the K Prelude or the K Standard Library)
+consists of several files which contain definitions that make working with K
+simpler.
```

Comment:
Maybe also add something like: These files can be found under `include/kframework/builtin/` in your installation directory.

Reply:
Added this: 

```
These files can be found under `include/kframework/builtin` in your K
installation directory, and can be imported with `requires "FILENAME"` (without
the path prefix).
```